packages/odyssey-storybook/.storybook/preview.js

```diff
@@ -17,7 +17,7 @@ export const parameters = {
     },
   },
   previewTabs: {
-    docs: { hidden: true },
+    "storybook/docs/panel": { index: -1 },
```

Comment:
can we configure this further to have docs navigation the default when clicking on links in the sb manager application? (sidebar UI nav)